goteoapi/projects/resources.py

```diff
@@ -54,6 +54,8 @@
     "description": fields.String,
     "type": fields.String,
     "amount": fields.Integer,
+    "units": fields.Integer,
+    "available_units": fields.Integer,
```

Comment:
This 2 fields need to be documented in the swagger docs:
`goteoapi/projects/swagger_specs/project_item.yml` section `id: ProjectReward`